homeworks/karolinavalko/4_exceptions_logging/json_to_do.py

```diff
@@ -0,0 +1,84 @@
+import logging
+import logging.config
+import os
+import json
+
+print(os.getcwd())
+
+logging.config.fileConfig(
+    r"c:\Users\tabal\OneDrive\ドキュメント\python_hu\homeworks\karolinavalko\4_exceptions_logging\logger_set_up.ini"
+)
+logger = logging.getLogger("my_logger")
+
+file_path = r"c:\Users\tabal\OneDrive\ドキュメント\python_hu\homeworks\karolinavalko\4_exceptions_logging\to_do.json"
+to_do_list = []
+display_menu = ["Add task", "View task", "Remove task", "Exit"]
+
+
+def read_file(file_path):
+    try:
+        with open(file_path, "r") as file:
+            logger.info("File had been opened in read mode.")
+            return json.load(file)
+    except FileNotFoundError as e:
+        logger.warning(f"The file {file_path} does not exist, creating one.")
+        write_list(file_path, [])
+        return []
+    except:
```

Comment:
szuper, hogy kezeled a váratlan eseményeket is, de itt adhatnál visszajelzést arról, hogy pontosan miért nem lehet beolvasni a file-t, ha hivatkozol is magára az Exception-re:

```
...
except Exception as e:
    logger.error(f"The file {file_path} could not be read: {e}.")
```